api/workAllocation2/util.ts

```diff
@@ -456,13 +456,15 @@ export function mapRoleCaseData(roleAssignment: RoleAssignment, caseDetail: Case
   return {
     assignee: roleAssignment.actorId,
     case_category: caseDetail.case_data.appealType,
+    case_type: caseDetail.case_type_id,
     case_id: caseDetail.id,
     case_name: caseDetail.case_data && caseDetail.case_data.caseName ? caseDetail.case_data.caseName : caseDetail.id,
     case_role: roleAssignment.roleName,
     role: roleAssignment.roleName,
     endDate: roleAssignment.endTime,
     id: roleAssignment.id,
     jurisdiction: caseDetail.jurisdiction,
+    jurisdictionId: caseDetail.jurisdiction,
```

Comment:
Why do we need `jurisdictionId` if it's just going to be assigned to `jurisdiction`

Reply:
jurisdictionId - IA
jurisdiction - Immigration And Asylum
We need both @kuda1992

Reply:
Understood the pr is failing unit tests thou.